src/client/api/event.c

```diff
@@ -90,6 +90,7 @@ daos_eq_lib_init()
 		D_GOTO(crt, rc);
 
 	eq_ref = 1;
+	ev_thpriv_is_init = false;
```

Comment:
minor, seems the static ev_thpriv_is_init should already be able to init as 0 by default when compiling

Reply:
Sure